api/percentile_agg.md

```diff
@@ -0,0 +1,283 @@
+# Percentile aggregate functions
+The functions related to percentile aggregates.
+
+## percentile_agg
+
+```sql
+percentile_agg(
+    value DOUBLE PRECISION
+) RETURNS UddSketch
+```
+
+This is the default percentile aggregation function. It uses the [UddSketch
+algorithm](https://github.com/timescale/timescale-analytics/blob/main/docs/uddsketch.md)
+with 200 buckets and an initial maximum error of 0.001. This is appropriate for
+most common use cases of percentile approximation. For more advanced use of
+percentile approximation algorithms,
+see [advanced usage](https://github.com/timescale/timescale-analytics/blob/main/docs/percentile_approximation.md#advanced-usage).
+This is the aggregation step
+of [two-step aggregates](https://github.com/timescale/timescale-analytics/blob/main/docs/two-step_aggregation.md),
```

Comment:
All of these links are going back to the github docs rather than internal, which may not be correct all the time?

Reply:
I think most of them are referring to content in the GH rather than in the docs. Let me know if you find one that's wrong, though.